secure_redis/views.py

```diff
@@ -0,0 +1,378 @@
+from __future__ import division
+
+from math import ceil
+
+from django.contrib import messages
+from django.contrib.admin.views.decorators import staff_member_required
+from django.http import Http404
+from django.shortcuts import redirect, render
+
+from redis.exceptions import ResponseError
+from rq import requeue_job, Worker
+from rq.exceptions import NoSuchJobError
+from rq.job import Job
+from rq.registry import (DeferredJobRegistry, FinishedJobRegistry,
+                         StartedJobRegistry)
+
+from django_rq.queues import get_connection, get_queue_by_index
+from django_rq.settings import QUEUES_LIST
+from . import settings
+
+_serializer = None
+
+
+def get_serializer():
```

Comment:
@GhalebKhaled This might be updated to use the singular object I referred to in my comment above for `decorators.oy`